saleor/app/management/commands/reinstall_apps.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any
+
+from django.core.management import BaseCommand
+
+from ...installation_utils import reinstall_app
+from ...models import App
+from ...tasks import install_app_task
+from ...types import AppType
+
+
+class Command(BaseCommand):
+    help = "Used to reinstall installed apps."
+
+    def handle(self, *args: Any, **options: Any):
+        # deactivate local apps
+        App.objects.filter(type=AppType.LOCAL).update(is_active=False)
```

Comment:
Why does it deactivate local apps?

Reply:
I may be wrong here, but as far as I remember we'd like to disable all apps initially, because its likely they will not work without reconfiguration. So, user will enable apps one by one, configuring them. 

For local apps - not sure if they will continue working if they are cloned at all?

But @przlada must confirm, I don't remember the details